mirrord/cli/src/teams.rs

```diff
@@ -0,0 +1,28 @@
+use tokio::process::Command;
+
+const MIRRORD_FOR_TEAMS_URL: &str = "https://mirrord.dev/docs/teams/introduction/";
+
+#[cfg(not(target_os = "macos"))]
+fn get_open_command() -> Command {
+    let mut command = Command::new("gio");
+    command.arg("open");
+    command
+}
+
+#[cfg(target_os = "macos")]
+fn get_open_command() -> Command {
+    Command::new("open")
+}
+
+/// Attempts to open mirrord for Teams introduction in the default browser.
+/// In case of failure, prints the link.
+pub async fn navigate_to_intro() {
+    match get_open_command().arg(MIRRORD_FOR_TEAMS_URL).output().await {
+        Ok(output) if output.status.success() => {}
+        other => {
+            tracing::trace!("failed to open browser, command result: {other:?}");
+            println!("mirrord failed to use a default browser");
```

Comment:
I wouldn't worry about printing this, it's not like an error the user would bother fixing

Reply:
Done